src/Transaction.js

```diff
@@ -50,6 +101,38 @@ class Transaction {
         }
     }
 
+    /**
+     * Create TxIns and send the current this.Tx object via RPC method with a returned hash and object with wait() for mining.
+     * @property {Function} sendTx - Create TxIns and send the current this.Tx object via RPC method
+     * @param {hex} [changeAddress=false] - Optional
+     * @param {hex} [changeAddressCurve=false] - Optional
+     * @param {Object} [UTXOIDs=[]] - Optional will be fetched if not provided
+     * @return {Promise<PendingTxObject>} Pending Tx Object
+     */
+    async sendWaitableTx(changeAddress, changeAddressCurve, UTXOIDs = []) {
+        try {
+            if (this.Tx.getTx()["Tx"]["Fee"] === 0) {
+                throw "No Tx fee added"
+            }
+            if (this.Tx.Vout.length <= 0) {
+                throw "No Vouts for transaction"
+            }
+            if (!this.Wallet.Rpc.rpcServer) {
+                throw 'No RPC to send transaction'
+            }
+            await this._createTxIns(changeAddress, changeAddressCurve, UTXOIDs);
+            await this.Tx._createTx();
+            console.log("\nSENDING: \n", JSON.stringify(this.Tx.getTx(), false, 2));
```

Comment:
Do we need this console log in prod?

Reply:
No, good catch thank you. leftover from the bug fix in the other PR haha :+1: